src/modules/dragAndDrop.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-use-before-define */
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable import/no-cycle */
```

Comment:
## [OPTIONAL]
- [ ] kindly you need to solve your linters instead of disabling them. Consider this in your next projects.

Reply:
Thank You for the review :pray: